test/quantization/test_quantize.py

```diff
@@ -765,6 +765,16 @@ def test_convtranspose_per_channel_fails_early(self):
             str(context.exception) ==
             'Per channel weight observer is not supported yet for ConvTranspose{n}d.')
 
+    @skipIfNoFBGEMM
+    def test_convtranspose_per_channel_qconfig_none(self):
+        r"""
+        Verifies that having qconfig==None for conv transpose does not crash
```

Comment:
This should work for any module correct? Would it be worth it to test for that too?

Reply:
I believe this is tested elsewhere for other modules, it just wasn't tested for ConvTranspose